runtime/syntax/man.vim

```diff
@@ -18,6 +18,11 @@ highlight default link manOptionDesc     Constant
 highlight default link manReference      PreProc
 highlight default link manSubHeading     Function
 
+if &filetype != 'man'
+  " Loaded by the location list TOC
+  finish
```

Comment:
This is for highlighting the qf window? That seems to really complicate the implementation. Is it really needed?